### PR TITLE
Hotfix/14

### PR DESCRIPTION
**Pull request by mmoufakkir1**
Login screen

### CODE REVIEW

File: my-pwa/.vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,2 @@
+{
+}
```
Comment (carlosap) [inline my-pwa/.vscode/settings.json]: Not sure what this is. this file should not be part of your PR

File: my-pwa/debug.log [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+[0221/132824.383:ERROR:process_info.cc(632)] range at 0x0, size 0x219 fully unreadable
+[0221/135024.596:ERROR:process_info.cc(632)] range at 0x1d577b6400000000, size 0x297 fully unreadable
+[0221/135024.597:ERROR:process_info.cc(632)] range at 0x1d577bc800000000, size 0x297 fully unreadable
+[0221/135024.597:ERROR:process_info.cc(632)] range at 0x0, size 0x297 fully unreadable
+[0221/163102.744:ERROR:process_info.cc(632)] range at 0x1b237b5600000000, size 0x169 fully unreadable
+[0221/163102.746:ERROR:process_info.cc(632)] range at 0x1b237bba00000000, size 0x169 fully unreadable
+[0221/163102.746:ERROR:process_info.cc(632)] range at 0x0, size 0x169 fully unreadable
+[0221/164546.340:ERROR:process_info.cc(632)] range at 0x6b617b5600000000, size 0x24c fully unreadable
+[0221/164546.340:ERROR:process_info.cc(632)] range at 0x6b617bba00000000, size 0x24c fully unreadable
+[0221/164546.340:ERROR:process_info.cc(632)] range at 0x0, size 0x24c fully unreadable
```
Comment (carlosap) [inline my-pwa/debug.log]: Not sure what this is. this file should not be part of your PR

File: my-pwa/package-lock.json [hunk 3]
```diff
@@ -0,0 +1,10986 @@
+{
```
Comment (carlosap) [inline my-pwa/package-lock.json]: this file should not be part of your PR

File: my-pwa/src/App.js [hunk 4]
```diff
@@ -35,6 +36,10 @@ class App extends Component {
   componentWillMount() {
     window.addEventListener('resize', this.updateDimensions);
     const onBoarded = store(keys.ONBOARDED);
+
+    console.log(' componentWillMount onBoarded ' + this.state.onboarded);
+    console.log(' componentWillMount loading ' + this.state.loading);
```
Comment (carlosap) [inline my-pwa/src/App.js]: please remove any console.logs for PRs

File: my-pwa/src/components/Intro/WelcomeScreen.js [hunk 5]
```diff
@@ -0,0 +1,270 @@
...
+import EmailIcon from 'material-ui-icons/Email';
+import Grid from 'material-ui/Grid';
+import { GoogleAPI, GoogleLogin, GoogleLogout, CustomGoogleLogin, CustomGoogleLogout } from 'react-google-oauth';
+import { createMuiTheme } from 'material-ui/styles';
+import * as globalReducers from '../../reducers/global';
+import * as globalActions from '../../actions/global';
+import * as keys from '../../constants/storageKeys';
+import App from '../../App';
+import { store } from '../../global'
+//import { GoogleLogin  } from 'react-google-login';
```
Comment (carlosap) [inline my-pwa/src/components/Intro/WelcomeScreen.js]: remove this please

File: my-pwa/src/components/Intro/WelcomeScreen.js [hunk 6]
```diff
@@ -0,0 +1,270 @@
...
+//import { GoogleLogin } from 'react-google-login-component';
```
Comment (carlosap) [inline my-pwa/src/components/Intro/WelcomeScreen.js]: remove this please

File: my-pwa/src/components/Intro/WelcomeScreen.js [hunk 7]
```diff
@@ -0,0 +1,270 @@
...
+
+
+const theme = createMuiTheme({
+  palette: {
+    primary: {
+      light: '#8e99f3',
```
Comment (carlosap) [inline my-pwa/src/components/Intro/WelcomeScreen.js]: please use material ui colors. you can import them by name and change contrast by digits
take a look at 
http://www.material-ui.com/#/customization/colors

File: my-pwa/src/components/Intro/WelcomeScreen.js [hunk 8]
```diff
@@ -0,0 +1,270 @@
...
+      main: '#5c6bc0',
+      dark: '#26418f',
+      contrastText: '#fff',
+    },
+    secondary: {
+      light: '#ff7961',
+      main: '#f44336',
```
Comment (carlosap) [inline my-pwa/src/components/Intro/WelcomeScreen.js]: same above

File: my-pwa/src/components/Intro/WelcomeScreen.js [hunk 9]
```diff
@@ -0,0 +1,270 @@
...
+  };
+
+  selectCategory(selectedCategory) {
+    this.setState({
+      selectedCategory,
+      isLoading: false,
+    });
+  }
+
+  signup(res, type) {
```
Comment (carlosap) [inline my-pwa/src/components/Intro/WelcomeScreen.js]: signUp?

File: my-pwa/src/components/Intro/WelcomeScreen.js [hunk 10]
```diff
@@ -0,0 +1,270 @@
...
+
+    let { isLogin, user } = this.props;
+
+    const isLoginPage = selectedCategory === 0;
+    const isSignUpPage = selectedCategory === 1;
+    const isPasswordRecovery = selectedCategory === 2;
+
+    console.log('isPasswordRecovery ' + isPasswordRecovery);
+    const responseGoogle = (response) => {
+      console.log("google console");
```
Comment (carlosap) [inline my-pwa/src/components/Intro/WelcomeScreen.js]: remove console.logs

File: my-pwa/src/components/Intro/WelcomeScreen.js [hunk 11]
```diff
@@ -0,0 +1,270 @@
...
+              <Grid item xs={12}>
+                <div style={styles.textField}
+                  style={styles.lineDivStyle}>
+                  <span
+                    style={styles.lineSpanStyle}> OR </span>
+                </div>
+              </Grid>
+              <Grid item xs={12}>
+                <GoogleAPI clientId="748666747267-sqglnr9ubnfrangqjpjrlcpmn5jv89mi.apps.googleusercontent.com"
+                  onUpdateSigninStatus={responseGoogleSigninStatus}
```
Comment (carlosap) [inline my-pwa/src/components/Intro/WelcomeScreen.js]: Remove key please. never show your clientid and save it github repo. key should be probably server or some hidden hard-coded config file. but never on gibhub.

File: my-pwa/src/reducers/global.js [hunk 12]
```diff
@@ -43,6 +43,15 @@ const initGlobalState = {
   selectedProjectId: '',
   selectedTasks: [],
   projects: [],
+  user: {
```
Comment (carlosap) [inline my-pwa/src/reducers/global.js]: user state should cover as much as the server responses. We can talk about this, take a look at the user response.
https://github.com/spartanappsolutions/Website/blob/master/api/SpartanServer/Models/User/UserResponse.cs#L5

File: my-pwa/src/store.js [hunk 13]
```diff
@@ -4,5 +4,6 @@ export default function configureStore(preloadedState) {
   return createStore(
     appReducer,
     preloadedState,
+    window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__(),
```
Comment (carlosap) [inline my-pwa/src/store.js]: not sure what this is. If this is not related to the project please remove.
If it is a tooling you have please do not submit this to code.

File: my-pwa/src/components/Intro/WelcomeScreen.js (hunk 5 above)
Comment (mmoufakkir1) [reply my-pwa/src/components/Intro/WelcomeScreen.js]: i can not remove this because it used within the app to bonus back

File: my-pwa/src/store.js (hunk 13 above)
Comment (mmoufakkir1) [reply my-pwa/src/store.js]: that piece of code allow to debug project state in chrome I will create a file to track that for me

File: my-pwa/src/reducers/global.js (hunk 12 above)
Comment (mmoufakkir1) [reply my-pwa/src/reducers/global.js]: this will be addressed in ticket #31
